packages/tradershub/src/components/Base/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+type TProps = {
+    alignment?: 'bottom' | 'left' | 'right' | 'top';
+    className?: string;
+    isVisible: boolean;
+    message: string;
+};
+
+const Tooltip: React.FC<React.PropsWithChildren<TProps>> = ({
+    alignment = 'left',
+    children,
+    className,
+    isVisible,
+    message,
+}) => {
+    return (
+        <div className={`relative w-max h-max ${className ?? ''}`}>
+            {children}
+            {isVisible && (
+                <div className={`absolute z-10 flex items-center transform-${alignment} ${className ?? ''}`}>
```

Comment:
same here

Reply:
it was coming from the wallets tooltip component scss
<img width="249" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/2c1a29ef-5931-491e-aeb0-e79b45eb72ed">

Reply:
I was talking about the classname